ftest.sh

```diff
@@ -148,17 +148,11 @@ DAOS_BASE=${SL_PREFIX%/install}
 if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
     -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
 sudo bash -c \"set -ex
-if [ \\\"\\\$(ulimit -l)\\\" != \\\"unlimited\\\" ]; then
-    echo \\\"*  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
-    echo \\\"root  soft  memlock  unlimited\\\" >> /etc/security/limits.conf
-    echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session-noninteractive
-    echo \\\"session required pam_limits.so\\\" > /etc/pam.d/common-session
-    echo \\\"session required pam_limits.so\\\" >> /etc/pam.d/login
-fi
 # allow core files to be generated
 if [ \\\"\\\$(ulimit -c)\\\" != \\\"unlimited\\\" ]; then
     echo \\\"*  soft  core  unlimited\\\" >> /etc/security/limits.conf
 fi
+prlimit --pid \$\$ --memlock=\"unlimited\"
```

Comment:
I don't think `prlimit` is the right approach here.  I think you had the right approach with `/etc/security/limits.conf`.

You have to think about what is actually being required by DAOS with this and how this will map to a production environment and I don't think `prlimit` is an even remotely a good mapping at all.